operator/src/main/java/oracle/kubernetes/operator/rest/RestConfigImpl.java

```diff
@@ -63,12 +58,12 @@ public int getInternalHttpsPort() {
 
   @Override
   public String getOperatorExternalCertificateData() {
-    return getCertificate(EXTERNAL_CERTIFICATE);
+    return TuningParameters.getInstance().getFileContents(EXTERNAL_CERTIFICATE);
```

Comment:
Should we move EXTERNAL_CERTIFICATE to OperatorConstants too? Seems odd that it is not defined in the same place as INTERNAL_CERTIFICATE

Reply:
I think EXTERNAL_CERTIFICATE, EXTERNAL_CERTIFICATE_KEY and INTERNAL_CERTIFICATE_KEY should all move to OperatorConstants.

Reply:
Actually, maybe instead of moving the constants to OperatorConstants, and writing yet more code that reads the certificate from the file, how about if AdminPodStepContext is somehow given a RestConfig instance - since RestConfig has methods that read the contents of the file - e.g.:

  /**
   * Gets the internal https port's certificate.
   *
   * @return base64 encoded PEM containing the certificate, or null if
   *     getOperatorInternalCertificateFile should be used instead to get the certificate.
   */
  String getOperatorInternalCertificateData();

Reply:
I refactored the code that reads the certificate from RestConfigImpl to TuningParameters as my first pass didn't look like it was going to be easy to pass the RestConfig to the AdminPodContext.  I can change it back and try and pass RestConfig if you prefer?

Reply:
Ok, I'll try and make change to give AdminPodContext a RestConfig instance.